packages/phpstan-rules/src/Rules/NoVoidGetterMethodRule.php

```diff
@@ -41,10 +41,10 @@ public function getNodeTypes(): array
     }
 
     /**
-     * @param ClassMethod $node
+     * @param ClassMethod $classMethod
      * @return string[]
      */
-    public function process(Node $node, Scope $scope): array
+    public function process(Node $classMethod, Scope $scope): array
```

Comment:
This actually breaks contract with parent interface. Better keep it that way to avoid possible miss-named bugs.

We should have a PHPStan rule for this :)

Reply:
you are talking about named-parameters BC breaks?

Reply:
Also, another potencial BC break is Rector matching param type by name.

Reply:
Just for the named arguments reference https://stitcher.io/blog/php-8-named-arguments

![image](https://user-images.githubusercontent.com/924196/126813567-36be39d1-5647-4327-a285-891d0bd156e6.png)

<br>

(I'm working on the PHPStan rule right now...)

Reply:
Here it is - https://github.com/symplify/symplify/pull/3429